csunplugged/templates/resources/treasure-hunt.html

```diff
@@ -0,0 +1,36 @@
+{% extends "resources/base-resource.html" %}
+
+<!-- Name of resource -->
+{% block name %}{{ resource.name }}{% endblock name %}
```

Comment:
Maybe we should talk about this when you go through all the templates (re your comment on the scratch PR)